### ISSUE TITLE
AppImage Shows Blank Page

### ISSUE THREAD

**Issue by luckycold** [cinnyapp/cinny-desktop#227]
### Describe the bug

Whether having logged in prior to 3.1.0 or trying to login for the first time on 3.1.0 only a blank page appears.

### Reproduction

1. Download 3.1.0 appimage
2. Run the appimage
3. Should show a blank page instead of login

### Expected behavior

Be able to see login and see rooms and dms.

### Platform and versions

```shell
1. OS: Fedora Linux
2. Cinny version: 3.1.0
3. Matrix homeserver: 1al.cc
4. Downloaded from: GitHub
```

**Comment by adevries17** [comment]
I also have this problem.
OS: Fedora 38
Cinny: 3.2.1
Downloaded from Github

Edit: version number

**Comment by kfiven** [comment]
Latest is 3.2.0 / 3.2.1 , have you checked with that?

**Comment by adevries17** [comment]
@kfiven Yes, just downloaded it. I was looking at what luckycold put as version and typed instead of the correct version number number I downloaded. I have downloaded 3.2.1 and it's a blank white screen

**Comment by adevries17** [comment]
I just built v3.2.1 on my machine and it works as expected. The flatpak of v3.2.1 also works for me. Maybe something in the prebuilt AppImage version is messed up?

**Comment by kfiven** [comment]
Only thing I could think of having such effect is the Ubuntu 20.04 and I can try bumping that to see if it solves the issue.

**Comment by Valiec** [comment]
I also have this issue on a Mac:

OS: macOS 12.6.2
Cinny: 3.2.1, 3.2.0, _or_ 3.1.0 (I tried all 3) Mac desktop app. Older versions might be affected too, I stopped trying at 3.1.0
Matrix Homeserver: Irrelevant, as I can't log into anything due to blank screen.
Downloaded from GitHub

Viewing the dev console shows a "SyntaxError: Invalid regular expression: invalid group specifier name" error.

**Comment by adevries17** [comment]
I ran from the terminal and it claims that `gstreamer element appsink not installed`. Not sure if that helps any
![image](https://github.com/cinnyapp/cinny-desktop/assets/63887471/fd2926b4-582d-4ff6-a8ac-045ccc7dc6bb)

**Comment by osousa** [comment]
I'm having the same issue, with "sudo dpkg -i .*.deb"

**Comment by ndpm13** [comment]
same issue here blank white screen on both Void Linux and Peppermint

**Comment by dermalikmann** [comment]
Seems like this problem popped up for me as well. I created another issue, as I am not using the AppImage, and also it now being 4.1.0. #329

**Comment by Maddest-Max** [comment]
Adding on for v4.2.0

### **Describe the bug**

Running the v4.2.0 appimage results in only a blank page appearing.

### **Reproduction**

1. Download v4.2.0 appimage
2. Run the appimage
3. Blank page appears as opposed to a login

### **Expected behavior**

Show login screen

### **Platform and versions**

1. OS: Manjaro
2. Cinny version: 4.2.0
3. Matrix homeserver: matrix.org
4. Downloaded from: GitHub

### **Additional context**

Running from the cli produces this:
```
Gtk-Message: 15:35:47.259: Failed to load module "xapp-gtk3-module"
Gtk-Message: 15:35:47.579: Failed to load module "xapp-gtk3-module"
```

**Comment by ChaosNicro** [comment]
Same here for AppImage 4.2.1. No output in terminal, but the dev-tools give: 
```
[Error] Unhandled Promise Rejection: SyntaxError: Invalid regular expression: invalid group specifier name
	RegExp (index-Nx5qNgae.js:27:4371)
	(anonymous function) (index-Nx5qNgae.js:27:4371)
	Module Code (index-Nx5qNgae.js:111:13543)
	evaluate
	moduleEvaluation
```
Looks like it's trying to parse a matrix.to component that no longer aligns with the RegEx.

**Comment by nac0n** [comment]
Same issue in 4.5.1

Running the appimage through terminal gives me this:

```
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
/usr/lib/x86_64-linux-gnu/gio/modules/libgiognomeproxy.so: undefined symbol: g_uri_is_valid
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgiognomeproxy.so
/usr/lib/x86_64-linux-gnu/libproxy/libpxbackend-1.0.so: undefined symbol: g_strv_builder_unref
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgiolibproxy.so

```

**Comment by Ricky-Tigg** [comment]
```
$ hostnamectl | sed -n '8p;10p' | awk '$1=$1'
Operating System: Fedora Linux 43.20260224.0 (Silverblue)
OS Support End: Wed 2026-12-02
```
Hello. It won't work, at least on Fedora. Evaluating the AppImage with `appimagelint` reveals it; it has not been tested against Fedora. So as expected, I also observe the application window with no content.

_v. 4.10.5_
```
$ ./Cinny_desktop-x86_64.AppImage 

** (cinny:3965): WARNING **: 09:28:29.342: webkit_settings_set_enable_offline_web_application_cache is deprecated and does nothing.
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...

** (cinny:3965): WARNING **: 09:28:54.484: atk-bridge: get_device_events_reply: unknown signature
```

**Comment by ArchEngel0** [comment]
Found a solution on:
Nobara 43 (Fedora essentially) Gnome, Wayland
Nvidia GPU
Cinny 4.10.5

Download the .deb
Extract using dpkg-deb
Navigate in .../usr/bin to the cinny executable
Terminal: `WEBKIT_DISABLE_DMABUF_RENDERER=1 ./cinny`
Success - No blank screen, no freezing, etc.

**Comment by nac0n** [comment]
This is no longer an issue for me in v4.10.5 AppImage. Can someone else confirm?

**Comment by Ricky-Tigg** [comment]
@ArchEngel0 | You were inspired. Until the fix is ​​created, this should serve as a workaround, on Fedora then.
